### ISSUE TITLE
Server Side version ?

### ISSUE THREAD

**Issue by p3rtinax** [YiannisBourkelis/Andama-Remote-Desktop#39]
Dear Yiannis,
First, thank you for this great application,
I wanna know is there any Server Side source code?
Which part has to run on Server to act as the Broker or gateway to connect clients to each other?

I have 10 pc on my workgroup, is it possible to host the server inside the office without using internet access ( self-host your app? )

**Comment by YiannisBourkelis** [comment]
@p3rtinax Yes there is a server side source code that acts as a proxy. It is the andama proxy located here: https://github.com/YiannisBourkelis/Andama-Remote-Desktop/tree/develop/src/AndamaProxy

Yes you can host andama on your private network without requiring an internet connection. You will need to compile andama proxy and run it in your server. Then compile Andama (better use the develop branch) and change the SERVER to point to your server hostname (line 930 in file clientserver.cpp https://github.com/YiannisBourkelis/Andama-Remote-Desktop/blob/d615e4e011c1c77d5c751bf14dd65a8bab3e3e8d/src/Andama/clientserver.cpp)
